Makefile

```diff
@@ -109,13 +107,21 @@ lint: lint-dependencies lint-all
 ############################################################
 # test section
 ############################################################
+KUBEBUILDER_DIR = /usr/local/kubebuilder
+KBVERSION = 3.2.0
+K8S_VERSION = 1.21.2
 
 test:
 	@go test ${TESTARGS} `go list ./... | grep -v test/e2e`
 
 test-dependencies:
-	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH).tar.gz | tar -xz -C /tmp/
-	sudo mv /tmp/kubebuilder_$(KBVERSION)_$(GOOS)_$(GOARCH) /usr/local/kubebuilder
+	sudo mkdir -p $(KUBEBUILDER_DIR)/bin/
+	curl -L https://github.com/kubernetes-sigs/kubebuilder/releases/download/v$(KBVERSION)/kubebuilder_$(GOOS)_$(GOARCH) -o /tmp/kubebuilder
+	sudo mv /tmp/kubebuilder $(KUBEBUILDER_DIR)/bin/
```

Comment:
Could you have curl directly download it to `$(KUBEBUILDER_DIR)/bin/kubebuilder` instead?

Reply:
Probably--let me check. That could also simplify the other command. I think the issue was repeatability, but we should probably log a message instead of overwriting files in `/usr/local/`.

Reply:
@mprahl Please take a look at my update. I check for files in `/usr/local/kubebuilder/bin/` and abort if files are found. Otherwise, it outputs files directly (without any `mv` directives).